### PR TITLE
[RW-618] Blrubenstein/api logging

### PR DESCRIPTION
**Pull request by blrubenstein**

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/exceptions/ExceptionUtils.java [hunk 1]
```diff
@@ -54,6 +54,7 @@ public static boolean isSocketTimeoutException(Throwable e) {
 
   public static RuntimeException convertFirecloudException(ApiException e) {
     log.log(e.getCode() >= 500 ? Level.SEVERE : Level.INFO, "Exception calling FireCloud", e);
+    log.log(Level.INFO, e.getResponseBody());
```
Comment (rushtong) [inline api/src/main/java/org/pmiops/workbench/exceptions/ExceptionUtils.java]: I would argue that the `INFO` lines here and on 56 (and below) should probably be `Level.WARN` at a minimum. If we're throwing an exception, I want these logs to stand out in Stackdriver.

File: api/src/main/resources/application.properties [hunk 2]
```diff
@@ -23,3 +23,5 @@ spring.datasource.max-idle=5
 spring.datasource.min-idle=1
 
 spring.jpa.hibernate.ddl-auto=none
+
+logging.level.org.pmiops.workbench=INFO
```
Comment (rushtong) [inline api/src/main/resources/application.properties]: Nit: newline

Comment (danqrodney) [reply api/src/main/resources/application.properties]: So I think the default log level in all environments right now is INFO already.

In any case, I'd remove this and log the above at WARNING / ERROR level.

File: api/src/main/java/org/pmiops/workbench/exceptions/ExceptionUtils.java (hunk 1 above)
Comment (danqrodney) [reply api/src/main/java/org/pmiops/workbench/exceptions/ExceptionUtils.java]: I would just make this a single log statement:

log.log(... "Exception calling FireCloud: " + e.getResponseBody(), e)

I'm fine with making it WARN for 4xx errors.

File: api/src/main/java/org/pmiops/workbench/exceptions/ExceptionUtils.java [hunk 3]
```diff
@@ -63,6 +64,7 @@ public static RuntimeException convertFirecloudException(ApiException e) {
   public static RuntimeException convertNotebookException(
       org.pmiops.workbench.notebooks.ApiException e) {
     log.log(e.getCode() >= 500 ? Level.SEVERE : Level.INFO, "Exception calling notebooks API", e);
+    log.log(Level.INFO, e.getResponseBody());
```
Comment (danqrodney) [inline api/src/main/java/org/pmiops/workbench/exceptions/ExceptionUtils.java]: See above, I'd combine these two log statements.

File: api/src/main/resources/application.properties (hunk 2 above)
Comment (blrubenstein) [reply api/src/main/resources/application.properties]: So, weirdly enough, without this line we weren't getting logs on our local. So I'm wondering if it broke somehow and we don't get default log level any more?

Comment (blrubenstein) [reply api/src/main/resources/application.properties]: I'll double check just to be sure.

Comment (blrubenstein) [reply api/src/main/resources/application.properties]: NM, it does work without the line, new version up now.

**Pull request merged**